StandaloneMmPkg/Drivers/StandaloneMmIplPei/MmFoundationHob.c

```diff
@@ -894,9 +937,11 @@ CreateMmFoundationHobList (
   //
   // BFV address for StandaloneMm Core
   //
-  HobLength = GetRemainingHobSize (*FoundationHobSize, UsedSize);
-  MmIplBuildFvHob (FoundationHobList + UsedSize, &HobLength, MmFvBase, MmFvSize);
-  UsedSize += HobLength;
+  if (!IsMmPlatformFvHobCreated (PlatformHobList, PlatformHobSize)) {

```

Comment:
Comments like "Skip to report FV that contains MmCore when Platform reports FV"

Reply:
updated